src/db/config/config.ts

```diff
@@ -0,0 +1,28 @@
+const dotEnv = require('dotenv');
```

Comment:
No need for this import here u can just call `require('dotenv).config()` or use es6 imports

Reply:
I get editor red squiggly lines in my editor that's why we gotta import everything. so this to me is flagged red `require('dotenv).config()`.